R/custom_disconnect_message.R

```diff
@@ -18,20 +19,20 @@
 #'
 #' @examples
```

Comment:
Is it worth showing an example for an EES publication too (so you have both non-EES and EES versions clear)

Reply:
Yep, removed those example lines just to test that it was working as I wanted with them set to NULL. Added a separate example now with them set as before.